skyportal/handlers/api/listing.py

```diff
@@ -0,0 +1,181 @@
+import re
+
+from baselayer.app.access import auth_or_token
+from ..base import BaseHandler
+from ...models import (
+    DBSession,
+    User,
+    Obj,
+    Listing,
+)
+
+
+class UserObjListHandler(BaseHandler):
+    @auth_or_token
+    def get(self):
+        """
+        ---
+            description: get all objects corresponding to a specific user and that match a list name.
+            parameters:
+            - in: query
+              name: user_id
+              required: true
+              type: string
```

Comment:
This should be a PATH arg

Reply:
Why should it be a path argument when there is a second argument as well? 
What is the advantage over just making both of them query?

Reply:
PATH arguments should be used whenever an argument is required, query arguments should always be optional.